internal/core/config.go

```diff
@@ -90,12 +89,20 @@ func (c DiscoveryConfiguration) FilterDomains(domains []KeystoneDomain) []Keysto
 	return result
 }
 
-// ServiceConfiguration describes a service that is enabled for a certain cluster.
-type ServiceConfiguration struct {
-	ServiceType db.ServiceType      `yaml:"service_type"`
-	PluginType  string              `yaml:"type"`
-	Area        string              `yaml:"area"`
-	Parameters  util.YamlRawMessage `yaml:"params"` // will be unmarshalled into the QuotaPlugin instance
+// LiquidConfiguration describes a service that is enabled for a certain cluster by means of a corresponding running liquid.
+// It holds configurations for how to deal with the service on project level (quota, usage, commitment) as well as cluster level (capacity).
+
+type LiquidConfiguration struct {
+	ServiceType db.ServiceType `yaml:"service_type"`
+	Area        string         `yaml:"area"`
+	// LiquidServiceType is a string that can be used to identify the liquid when generating a liquidapi.NewClient.
```

Comment:
This description is rather vague:
```suggestion
	// LiquidServiceType is the service type under which the liquid is registered in the Keystone catalog.
```